extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -716,7 +716,8 @@ export class AmpA4A extends AMP.BaseElement {
       dev().assert(getMode().localDev || !endsWith(serviceName, '-dev'));
       const url = signingServerURLs[serviceName];
       if (url) {
-        return xhrFor(this.win).fetchJson(url, {mode: 'cors', method: 'GET'})
+        return xhrFor(this.win).fetchJson(
+          url, {mode: 'cors', method: 'GET', disableAmpSourceOrigin: true})
```

Comment:
Reformat object literal to multiple lines. Should this be also `credentials: 'omit'`?